tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -136,6 +136,50 @@ public void ExcludeLaunchProfileOmitsBindings()
             "Service has no bindings because they weren't populated from the launch profile.");
     }
 
+    [Fact]
+    public void ProjectResourceEmitsForwardedHeadersEnvironmentVariableByDefault()
```

Comment:
Move these tests into ProjectTests. I want to delete this test class eventually.

Reply:
I deleted these tests and added new ones in `ProjectResourceTests` more inline with how they're done there.